src/main/java/com/example/employeecrudapplication/validator/EmployeeValidator.java

```diff
@@ -0,0 +1,45 @@
+package com.example.employeecrudapplication.validator;
+
+import com.example.employeecrudapplication.exception.EmployeeValidationException;
+import com.example.employeecrudapplication.model.EmployeeDto;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EmployeeValidator {
+
+    public void validateEmail(String email) {
+        if (
+                StringUtils.isBlank(email)
+                || (!email.contains("@"))
+                || (!email.contains("."))
+        ) {
+            throw new EmployeeValidationException("Employee Email is not present in the request");
+        }
+    }
+
+    public void validateFields(EmployeeDto employee) {
+        if (employee == null) {
+            throw new EmployeeValidationException("Null Pointer Exception! Employee request is null");
```

Comment:
Change message to "Employee not provided"

Reply:
![image](https://user-images.githubusercontent.com/100875409/219569310-897427a9-930a-4dd9-a69e-16de4d1c6d36.png)

Reply:
Method should be void. Thus, remove return statement

Reply:
I don't understand what I should do yet, but I have just made it like this:
![image](https://user-images.githubusercontent.com/100875409/219571862-4a4fbde4-9d2c-487f-a430-4aa5bdb3e892.png)
I will try reading https://www.baeldung.com/spring-response-status-exception as you suggested